pkgs/applications/audio/infamousPlugins/default.nix

```diff
@@ -2,13 +2,14 @@
 
 stdenv.mkDerivation rec {
   pname = "infamousPlugins";
-  version = "0.3.0";
+  # Fix build with lv2 1.18: https://github.com/ssj71/infamousPlugins/commit/4c7275b1fa8ea3296446421cbd29ec2df66588c0
+  version = "unstable-2020-04-29";
 
   src = fetchFromGitHub {
     owner = "ssj71";
     repo = "infamousPlugins";
-    rev = "v${version}";
-    sha256 = "1r72agk5nxf5k0mghcc2j90z43j5d9i7rqjmf49jfyqnd443isip";
+    rev = "4c7275b1fa8ea3296446421cbd29ec2df66588c0";
+    sha256 = "0l5f39cjcyfkh20zi2d9h7bdy43wz4qyirhj6xc0fcq2r537461r";
   };
 
```

Comment:
Is it not sufficient to add `https://github.com/ssj71/infamousPlugins/commit/06dd967b4736ea886dc1dc07f882cb1563961582.patch` to `patches` using `fetchpatch`?

Reply:
That works.